### DISCUSSION TITLE
Missing Order

### DISCUSSION THREAD

**Discussion by JCOterinoImpackta** [amzn/selling-partner-api-models#3263] [General] [answered]
I have a question hopefully someone can help please.
I am using the api to get orders and addresses and all is working fine in the Spanish MarketPlace, but when someone from Italy buy an article in this marketplace the order is not coming back in the response. I can see the order in my shop.
I tried to get the order by number and no luck, and neither is coming with the other order in the response, I also tried to get adress of this order and YES it is coming back OK. so I don´t know why is this happening.
¿Cand someone point me how to get this order?

**Comment by JCOterinoImpackta** [comment, marked as answer]
Please forgive the question.
Cause definitely this is not the site to ask for.
Thanks

**Comment by RoanOak127** [comment]
For you or anyone else who has a similar question: The Europe region (eu-west-1)  has multiple marketplaces, I know in the solution that I'm using I need to add the rest of the European countries to the list of approved marketplaces for my customer in order to see them properly in our system. However you're using SPAPI you might also just need to update your list of Marketplaces.

**Reply by JCOterinoImpackta** [reply]
Thanks for the reply but what do you mean with "add the rest of the European Countries"??? and most important where or how to update the list of MarketPlace???  What do you mean, with that knowing that my customer is just authorized for Spanish Marketplace.
